tool/dart_site/lib/src/commands/build.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+
+import '../utils.dart';
+
+final class BuildSiteCommand extends Command<int> {
+  static const String _releaseFlag = 'release';
+
+  BuildSiteCommand() {
+    argParser.addFlag(
+      _releaseFlag,
+      defaultsTo: false,
```

Comment:
This flag defaults to false

Reply:
Good point!

So the optional parameter for a flag that defaults to false, also defaults to false 🤣 

Do you mind if I keep these though? It's then easier for someone coding in these files to identify the flag defaults to `false` rather than `true` or anything else, especially if they're not super familiar with `package:args`.

Reply:
Yeah, that's fair!